src/routes/about.tsx

```diff
@@ -0,0 +1,101 @@
+import Page from '@/components/page'
+import { css } from 'linaria'
+
+const pageStyle = css`
+  text-align: center;
+  padding: 0 1rem;
+  margin: auto;
+  max-width: 50rem;
+  min-width: 70%;
+`
+
+const headerStyle = css`
+  margin-bottom: 0;
+  margin-top: 1em;
+`
+
+const informationStyle = css`
+  font-size: 22px;
+`
+
+const sourceStyle = css`
+  padding: 1rem 0;
+  font-family: monospace;
+
+  & h3 {
+    margin: 0.3rem;
+  }
+
+  & p {
+    margin: 0.2rem;
+  }
+`
+
+const AboutPage = () => {
+  return (
+    <Page name="About" back="/" class={pageStyle}>
+      <h1 class={headerStyle}>Welcome to Peregrine!</h1>
+      <section class={informationStyle}>
+        <p>
+          <strong>
+            Peregrine is a completely free, open-source scouting app for FRC
+            competitions.
+          </strong>{' '}
+          The frontend is written in TypeScript and the backend is written in
+          Go. Peregrine was originally built by students on team 2733 Pigmice in
+          2017 and redesigned in the autumn of 2018. Development is ongoing to
+          make the website better than ever. Peregrine has been used by 15 teams
+          and counting to simplify scouting at FRC competitions.
+        </p>
+        <p>
+          <strong>
+            Peregrine handles all of the calculations involved in scouting
+          </strong>
+          , allowing team members to spend time analyzing data and making
+          decisions instead of crunching numbers. It shows an analysis table
+          where team members can compare individual statistics or overall
+          performance between teams and has pages for each team at each event
+          and the matches each team will play in, easily allowing pit crews to
+          know when they&rsquo;re up next. All event data and some match scoring
+          data is pulled from The Blue Alliance.
+        </p>
+        <p>
+          <strong>It&rsquo;s really easy to get started with Peregrine.</strong>{' '}
```

Comment:
@varCepheid To avoid the annoying spaces-in-embedded-expressions things, we can use an embedded string for the whole text. It would also handle the HTML escaping automatically for you if you do that.

Like this:

```
<strong>{"It's really easy to get started with Peregrine."}</strong>
```
and

```
{" Have a team captain or coach send an email to "}
```

I think that makes it a little easier to read the code and it shouldn't affect the output.

I'd suggest making this change throughout the whole about text.